### PR TITLE
Fixed for new server

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: nginx/sites-available/backend.break-out.org.conf [hunk 1]
```diff
@@ -10,13 +10,13 @@ server {
...
 server {
-    listen        443 ssl spdy;
+    listen        443 ssl;
 
     server_name   backend.break-out.org;
 
     ssl on;
-    ssl_certificate     /etc/letsencrypt/live/backend.break-out.org/fullchain.pem;
-    ssl_certificate_key /etc/letsencrypt/live/backend.break-out.org/privkey.pem;
+    ssl_certificate     /etc/letsencrypt/live/anmeldung.break-out.org/fullchain.pem;
```
Comment (nerdsupremacist) [inline nginx/sites-available/backend.break-out.org.conf:18]: are you sure?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # breakout-loadbalancer
 A loadbalancer config for all services
+
+## Setup on Ubuntu
+
+ - install nginx and [certbot](https://certbot.eff.org/lets-encrypt/ubuntubionic-nginx)
```
Comment (nerdsupremacist) [inline README.md:6]: Awesome! I love it 👍

File: nginx/sites-available/break-out.org.conf [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+
+server {
+    listen        443 ssl;
+
+    server_name   break-out.org;
+
+    client_max_body_size 2G;
+
+    ssl on;
+    ssl_certificate     /etc/letsencrypt/live/anmeldung.break-out.org/fullchain.pem;
```
Comment (nerdsupremacist) [inline nginx/sites-available/break-out.org.conf:22]: again are you sure?

File: nginx/sites-available/feedback.break-out.org.conf [hunk 4]
```diff
@@ -11,13 +11,13 @@ server {
...
 server {
-    listen        443 ssl spdy;
+    listen        443 ssl;
 
     server_name   feedback.break-out.org;
 
     ssl on;
-    ssl_certificate     /etc/letsencrypt/live/feedback.break-out.org/fullchain.pem;
-    ssl_certificate_key /etc/letsencrypt/live/feedback.break-out.org/privkey.pem;
+    ssl_certificate     /etc/letsencrypt/live/anmeldung.break-out.org/fullchain.pem;
```
Comment (nerdsupremacist) [inline nginx/sites-available/feedback.break-out.org.conf:19]: wait? why is everything using the same certificate?

**Comment by thepiwo** [comment]
@nerdsupremacist we use the same certificate for all sites now, when we initially set up the loadbalancer it was not possible for letsecrypt to issue certificates with multiple names, which now is the default, as the certbot automatically chooses one of the domain names as certificata name all of the certificates are included in this one file.

**Pull request merged**